indexing-service/src/main/java/io/druid/indexing/overlord/RemoteTaskRunner.java

```diff
@@ -872,4 +880,57 @@ private void taskComplete(
     // Notify interested parties
     taskRunnerWorkItem.setResult(taskStatus);
   }
+
+  public List<ZkWorker> findLazyworkers(Predicate<ZkWorker> isLazyWorker, int excessWorkers)
```

Comment:
maybe we can rename this to `markWokersLazy` to indicate this actually modifies the state of lazy workers.
Also, renaming `excessWorkers` to `maxWorkers` would clarify this is the maximum number of workers it will mark as lazy